### PR TITLE
sp_load_solrtreatmententrylevel

### PR DESCRIPTION
**Pull request by carresemata**
Translated sp_load_solrtreatmententrylevel:
Validation:
* Columns seem to be the same
* The column ForMarketViewLoad is 0 and 1 in sql server and True and False in snowflake

### CODE REVIEW

File: migration_original/ODS1Stage/tables/Show/SOLRTreatmentEntryLevel/spu_translated_SOLRTreatmentEntryLevel.sql [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    EXECUTE AS CALLER
+    AS  
+
+DECLARE
+
+---------------------------------------------------------
+--------------- 0. Table dependencies -------------------
+---------------------------------------------------------
+    
+-- SOLRTreatmentEntryLevel depends on : Show.SOLRTreatmentEntryLevel, Base.SpecialtyToCondition, Base.TreatmentLevel, Base.SpecialtyToProcedureMedical, Base.Specialty, and Base.MedicalTerm
```
Comment (ggandia-xtillion) [inline migration_original/ODS1Stage/tables/Show/SOLRTreatmentEntryLevel/spu_translated_SOLRTreatmentEntryLevel.sql:15]: One row per table for readability
```suggestion
-- SOLRTreatmentEntryLevel depends on :
---- Show.SOLRTreatmentEntryLevel
---- Base.SpecialtyToCondition
---- Base.TreatmentLevel
---- Base.SpecialtyToProcedureMedical
---- Base.Specialty
---- Base.MedicalTerm
```

**Review by ggandia-xtillion — CHANGES_REQUESTED** [review]

File: migration_original/ODS1Stage/tables/Show/SOLRTreatmentEntryLevel/spu_translated_SOLRTreatmentEntryLevel.sql [hunk 2]
```diff
@@ -0,0 +1,124 @@
+
+-- 1. spuSOLRTreatmentEntryLevel (validated in snowflake)
+CREATE OR REPLACE PROCEDURE SHOW.SP_LOAD_SOLRTREATMENTENTRYLEVEL()
```
Comment (omargoldpr) [inline migration_original/ODS1Stage/tables/Show/SOLRTreatmentEntryLevel/spu_translated_SOLRTreatmentEntryLevel.sql:3]: Add database name to the procedure name.
```suggestion
CREATE OR REPLACE PROCEDURE ODS1STAGE.SHOW.SP_LOAD_SOLRTREATMENTENTRYLEVEL() 
```

**Review by omargoldpr — CHANGES_REQUESTED** [review]
Why is spu_original_SOLRProvider_Freeze.sql part of this PR?

File: migration_original/ODS1Stage/tables/Show/SOLRTreatmentEntryLevel/spu_translated_SOLRTreatmentEntryLevel.sql (hunk 1 above)
Comment (carresemata) [reply migration_original/ODS1Stage/tables/Show/SOLRTreatmentEntryLevel/spu_translated_SOLRTreatmentEntryLevel.sql]: Done

File: migration_original/ODS1Stage/tables/Show/SOLRTreatmentEntryLevel/spu_translated_SOLRTreatmentEntryLevel.sql (hunk 2 above)
Comment (carresemata) [reply migration_original/ODS1Stage/tables/Show/SOLRTreatmentEntryLevel/spu_translated_SOLRTreatmentEntryLevel.sql]: Fixed it

**Pull request merged**